src/Pos/Launcher/Runner.hs

```diff
@@ -173,6 +173,7 @@ runRawRealMode res np@NodeParams {..} sscnp listeners outSpecs (ActionSpec actio
   where
     LoggingParams {..} = bpLoggingParams npBaseParams
 
+-- TODO: remove this function?
```

Comment:
Because it uses `sysStart`. It's not clear for me what is _leftover_ so I'm asking about every `sysStart`